src/libraries/Microsoft.Extensions.Caching.Memory/src/Microsoft.Extensions.Caching.Memory.csproj

```diff
@@ -7,16 +7,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Reference Include="Microsoft.Extensions.Caching.Abstractions" />
-    <Reference Include="Microsoft.Extensions.DependencyInjection.Abstractions" />
-    <Reference Include="Microsoft.Extensions.Logging.Abstractions" />
-    <Reference Include="Microsoft.Extensions.Options" />
-    <Reference Include="Microsoft.Extensions.Primitives" />
-  </ItemGroup>
-
-  <ItemGroup Condition="$(TargetFramework.StartsWith('net4'))">
-    <Reference Include="mscorlib" />
-    <Reference Include="System.Core" />
+    <ProjectReference Include="$(LibrariesProjectRoot)Microsoft.Extensions.Caching.Abstractions\src\Microsoft.Extensions.Caching.Abstractions.csproj" />
+    <ProjectReference Include="$(LibrariesProjectRoot)Microsoft.Extensions.DependencyInjection.Abstractions\src\Microsoft.Extensions.DependencyInjection.Abstractions.csproj" />
```

Comment:
@ViktorHofer can you add these project references to the .sln files as well ?

Reply:
I would need to do that for the ref ones as well which results in an explosion of each solution file. The workaround is to make sure that you do a CLI restore first (which is indicated by the error in VS) which also works around the other NuGet restore issue that we are currently tracking. So for the sake of this PR I wouldn't want to add them and in a follow-up convert all slns to slnfs and have a root solution file.

Reply:
When using the -vs switch, the CLI restore is incremental so that's covered.